### ISSUE TITLE
Succesfully added a Nextion Display to your Espradio

### ISSUE THREAD

**Issue by Manu-J** [Edzelf/Esp-radio#139]
Thanks for making your code an opensource, I just modified it to interact it with my Nextion display,

More details on my project find [Here](https://www.youtube.com/watch?v=1q8UP5HhdpM&t=36s). Once again thanks a lot.

https://www.youtube.com/watch?v=1q8UP5HhdpM&t=36s

![snapshot_11](https://user-images.githubusercontent.com/39552389/41501389-737d7cb4-719b-11e8-9a80-3719815acc91.png)

**Comment by Edzelf** [comment]
Looks good!

**Comment by Manu-J** [comment]
Yes I am using the Rx tx pins of Nodemcu.I initially tried to use software serial pins but it was throwing a lot of issues..but when I used like this no issues,it can communicate with Nextion as well as with PC serial port.The only problem is that the debug output will show also the Nextion display commands. The only thing to remeber is that to make sure that the Nextion and Nodemcu baudrates are set for same value. For this project I set the baudrates as 115200 ans it worked like charm.
